va/validation-authority.go

```diff
@@ -511,8 +513,10 @@ func (va *ValidationAuthorityImpl) validateHTTP01(identifier core.AcmeIdentifier
 		return challenge, err
 	}
 
+	payload := strings.TrimRight(string(body), "\n\t ")
```

Comment:
also, please make this a const called whitespaceCutset so we can drop docs and a citation to the spec for it.